Balanced-BT.cpp

```diff
@@ -0,0 +1,35 @@
+// Time Complexity : O(nlogn)
```

Comment:
Time: O(n^2). Height is recalculated for every node, tree traversed repeatedly. Please try the bottom up approach as well.